Source/WebCore/dom/Range.h

```diff
@@ -52,6 +52,9 @@ class Range final : public AbstractRange, public CanMakeWeakPtr<Range> {
     bool collapsed() const final { return m_start == m_end; }
     WEBCORE_EXPORT Node* commonAncestorContainer() const;
 
+    void resetDidChange() { m_didChange = false; }
+    bool didChange() const { return m_didChange; }
```

Comment:
These functions are specific for highlights so perhaps we should communicate that.
e.g. resetDidChangeHighlight & didChangeHighlight.

Reply:
Got it, will change!